src/ToDoList.js

```diff
@@ -86,17 +85,16 @@ const ToDoList = () => {
   };
 
   const filterTasks = () => {
-    if (taskState === "completed")
-      return newTaskValue.filter((task) => task.completed);
+    if (taskState === "isCompleted")
+      return newTaskValue.filter((task) => task.isCompleted);
```

Comment:
please place the `return` code in the same line as the `if` statement for increased readability.
another acceptable option is to wrap it inside curly brackets, but usually for single lines,
we prefer the first option.

Reply:
When I save the changes it automatically lowers it to the next line. 
Also, the code was changes acording to a different comment:
```
const filterTasks = () => {
    if (taskStateFilter === FilterByTaskStateOption.COMPLETED)
      return newTaskValue.filter((task) => task.isCompleted);
    if (taskStateFilter === FilterByTaskStateOption.TODO)
      return newTaskValue.filter((task) => !task.isCompleted);
    return newTaskValue;
  };
```